tests/crudRoutes.js

```diff
@@ -0,0 +1,69 @@
+const request = require('supertest');
+const express = require('express');
+
+const app = require("../server/server.js");
+
+//Create
+describe('POST /writeReview/:productid', () => {
```

Comment:
Do posts take a productid param? I haven't been doing this, but I'm curious how it could be used.

Reply:
I think they need to for Reviews since the review needs to be associated with a specific product. But let's discuss tomorrow. I'm sure there's a better way to indicate which product the user is reviewing.